context_test.go

```diff
@@ -41,6 +41,25 @@ func TestRespond(t *testing.T) {
 	}
 }
 
+func TestStatus(t *testing.T) {
+	c := NewContext()
+	data := map[string]string{"name": "jim"}
+	r := c.Status(201, data)
+	if r.StatusCode != 201 {
+		t.Errorf("Status code not set to 201 (%d)", r.StatusCode)
+	}
+	if r.Error != nil {
```

Comment:
These same types of checks seem to be throughout the tests, might be good to create a helper for them to be D.R.Y.